src/app/features/animals/components/animals-list/animals-list.component.html

```diff
@@ -1,4 +1,60 @@
-<h1>Animals</h1>
+<div class="row">
+  <div class="col">
+    <h1>Animals</h1>
+  </div>
+  <div class="col text-end">
+    <button class="btn btn-success" routerLink="/animals/add">Add new</button>
+  </div>
+</div>
 
-<!-- TODO: Add table with animals -->
-<p>Coming soon...</p>
+<ul class="nav nav-tabs mb-4">
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('table')" [class.active]="activeTab === 'table'">Table</a>
```

Comment:
Použil si taby s nastavením `activeTab`, ale namiesto toho by bolo lepšie použiť `routerLink` a nastaviť pre každý tab vlastnú URL.

```suggestion
    <a class="nav-link" routerLink="/animals/table" routerLinkActive="active">Table</a>
```

Reply:
Tak isto nezabudni upravit aj 'routes' potom a pouzit children roots.